yml/dp02_dc2.yaml

```diff
@@ -8183,232 +8183,6 @@ tables:
     mysql:datatype: FLOAT
     description: 1-sigma uncertainty on y position
     fits:tunit: pixel
-- name: ForcedSourceOnDiaObject
```

Comment:
Would you want to leave this in, but commented out (#), as was done for the match tables in this file (just to ease reference and also to make later resuscitation easier?)

Reply:
Ok, and that's easier if there's anything else we need to change while it's commented-out.